source/Activated.js

```diff
@@ -35,6 +35,13 @@ const Activated = ({
     setIsUnloading(true)
   }, [])
 
+  const handleEscClick = useCallback(e => {
+    e.preventDefault()
+    if (e.keyCode === 27) {
```

Comment:
As `keyCode` is deprecated on MDN (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), we should probably check for `e.key === 'Escape'` first!
```js
if (e.key === 'Escape' || e.keyCode === 27) {
```

Reply:
Applied!